TenancyInformationApi/V1/Infrastructure/UhContext.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace TenancyInformationApi.V1.Infrastructure
+{
+    public class UhContext : DbContext
+    {
+        public UhContext(DbContextOptions options) : base(options)
+        {
+            var tenancyAgreements = UhTenancyAgreements
+                .Include(t => t.UhAgreementType)
+                .Include(t => t.UhTenureType)
+                .ToList();
```

Comment:
Is this bit of code used for anything?

Reply:
It is pulling everything from the tenancy agreement table, so is quite an expensive database operation to make

Reply:
The relation in EF wasn't working until I performed eager loading like this. I'm definitely interested in better solutions as, like you say, this is really expensive.